index.js

```diff
@@ -4,11 +4,19 @@ function getBaseNameFromUrl(url) {
 
 var injectTemplates = function(templates) {
     templates.forEach(function(tpl) {
-        var s = document.createElement('script');
-        s.id = tpl.id;
-        s.innerHTML = tpl.content;
-        s.type = 'text/html';
-        document.head.appendChild(s);
+        if (tpl.content.trim().slice(0, 7) === '<script') {
```

Comment:
This check assumes that all cases where you have nested templates all have the nested template on the opening string. I'm not sure if that really covers all the cases?